ClothesApp/Application/Services/ReviewService.cs

```diff
@@ -14,18 +15,19 @@ public class ReviewService : IReviewService
     private readonly IProductsRepository _productsRepository;
 
     private readonly IUserRepository _userRepository;
-    
+
     private readonly IMapper _mapper;
 
-    public ReviewService(IReviewsRepository reviewsRepository, IMapper mapper, IProductsRepository productsRepository, IUserRepository userRepository)
+    public ReviewService(IReviewsRepository reviewsRepository, IMapper mapper, IProductsRepository productsRepository,
+        IUserRepository userRepository)
     {
         _reviewsRepository = reviewsRepository;
         _mapper = mapper;
         _productsRepository = productsRepository;
         _userRepository = userRepository;
     }
 
-    public async Task<ReviewDto> Add(ReviewInputDto reviewInputDto)
+    public async Task<ReviewDto> Add(ReviewInputDto reviewInputDto, Claim userClaimId)
```

Comment:
Why not just pass userId, I doubt service should know about claims and other stuff